packet/loadbalancers.go

```diff
@@ -41,6 +41,9 @@ func newLoadBalancers(client *packngo.Client, projectID, facility string, config
 	if len(cmparts) >= 2 {
 		configmapnamespace, configmapname = cmparts[0], cmparts[1]
 	}
+	if configmap == "none" {
+		configmapname = configmap
+	}
```

Comment:
So this now sets the `configmapnamespace="none"` and `configmapname="none"`?

Reply:
So is `"none"` a way to say, "do all of your loadbalancer stuff, but do not do metallb stuff"? Now we have 2 modes:

* `disabled` - do not do LB
* not-`disabled` (either `""` which defaults to `"metallb-system:config"`, or something else explicit) - do LB and metallb config

This would add a third mode:

* `none` - do all of the LB stuff _except_ for the metallb configmap

Is that it?